pincer/middleware/interaction_create.py

```diff
@@ -88,18 +89,37 @@ async def interaction_handler(
     """
     self.throttler.handle(context)
 
-    defaults = {param: None for param in get_params(command)}
+    sig, params = get_signature_and_params(command)
+
+    defaults = {key: value.default for key,
+                value in sig.items() if value.default is not _empty}
     params = {}
```

Comment:
Seeing how `params` is not getting used until getting "manually" updated, I'd do the following
```suggestion
    sig, _ = get_signature_and_params(command)

    defaults = {key: value.default for key,
                value in sig.items() if value.default is not _empty}
    params = {}
```